Automattic/wp-calypso
# Tailored Flows: /start clean up

## Proposed Changes

The first release of Newsletter and LIB flow was using `/start`, [now it is not anymore](https://github.com/Automattic/wp-calypso/pull/74411). This is a cleanup PR removing LIB and mainly Newsletter code from `/start` since it is not being used anymore.


## Testing Instructions
This is a bit tricky since it is hard to keep track of all the flows using `/start`

I have in mind the following

Case 1:

- Test NL and/or LiB flow won't make much sense since it is not using /start anymore, but worths a second check, although I am not expecting issues there.

Case 2: Test `/start` which means create a flow from scratch.

- Pull and run this branch or use calypso live links
- Navigate to `/start` for instance `{calypsoLiveLink or Localhost}/start`, that should render the domains step since there's no specific flow.
- Try to complete the flow.
- There should be no console erros
- There should be no Newsletter or LiB copies on domains and plans step, copies should be more "generic"
